sh.loft.Devpod.yaml

```diff
@@ -0,0 +1,50 @@
+id: sh.loft.Devpod
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: DevPod
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
+  - --share=network
+  - --socket=ssh-auth
+  - --socket=gpg-agent
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/loft-sh/devpod/releases/download/v0.6.10/DevPod_0.6.10_amd64.deb
+        sha256: d43738efa60735cc4ad6b63285f9e449534831ba9a2c7ac2c72cb682fb26f4b5 # This is required if you are using a remote source
+        only-arches: [x86_64] #This source is only used on x86_64 Computers
```

Comment:
so what happen when we build the aarch64 package?

why not building this from source?

Reply:
We build our desktop app and CLI in our main repo https://github.com/loft-sh/devpod, we currently do not support linux arm64 for the desktop app as we do not yet have the CI set up to cross compile. So for now we only support amd for our desktop app on linux.

Reply:
so, again, what happen when the aarch64 package is built?

Reply:
still doesn't explain why you don't build from source either.

Reply:
Nonetheless an aarch64 package is still built.


https://docs.flathub.org/docs/for-app-authors/maintenance#flathubjson

Reply:
Added flathub.json :+1: